<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>sort_window</class>
 <widget class="QDialog" name="sort_window">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>568</width>
    <height>421</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Dialog</string>
  </property>
  <property name="styleSheet">
   <string notr="true">background-color: rgb(54, 54, 54);</string>
  </property>
  <widget class="QGroupBox" name="groupBox">
   <property name="geometry">
    <rect>
     <x>360</x>
     <y>10</y>
     <width>201</width>
     <height>401</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">background-color: rgb(85, 85, 85);</string>
   </property>
   <property name="title">
    <string/>
   </property>
   <widget class="QComboBox" name="comboBox_selectalgo">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>40</y>
      <width>111</width>
      <height>22</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">background-color: rgb(243, 243, 243);
selection-background-color: rgb(83, 83, 83);

</string>
    </property>
   </widget>
   <widget class="QLabel" name="label_switchalgo">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>20</y>
      <width>91</width>
      <height>16</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">font: 63 9pt &quot;Yu Gothic UI Semibold&quot;;
color:white;</string>
    </property>
    <property name="text">
     <string>Switch algorithm</string>
    </property>
   </widget>
   <widget class="QLabel" name="label_selectinputoption">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>70</y>
      <width>91</width>
      <height>16</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">font: 63 9pt &quot;Yu Gothic UI Semibold&quot;;
color:white;</string>
    </property>
    <property name="text">
     <string>Select Input Option</string>
    </property>
   </widget>
   <widget class="QComboBox" name="comboBox_selectipop">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>90</y>
      <width>111</width>
      <height>22</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">background-color: rgb(243, 243, 243);
selection-background-color: rgb(83, 83, 83);

</string>
    </property>
   </widget>
   <widget class="QLabel" name="label_customip">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>120</y>
      <width>71</width>
      <height>16</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">font: 63 9pt &quot;Yu Gothic UI Semibold&quot;;
color:white;</string>
    </property>
    <property name="text">
     <string>Custom Input</string>
    </property>
   </widget>
   <widget class="QLabel" name="label_seqlength">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>166</y>
      <width>81</width>
      <height>20</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">font: 63 9pt &quot;Yu Gothic UI Semibold&quot;;
color:white;</string>
    </property>
    <property name="text">
     <string>Sequence Length</string>
    </property>
   </widget>
   <widget class="QLineEdit" name="lineEdit_customip">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>140</y>
      <width>113</width>
      <height>20</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">background-color: rgb(243, 243, 243);
selection-background-color: rgb(83, 83, 83);

</string>
    </property>
   </widget>
   <widget class="QLineEdit" name="lineEdit_seqlength">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>190</y>
      <width>113</width>
      <height>20</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">background-color: rgb(243, 243, 243);
selection-background-color: rgb(83, 83, 83);

</string>
    </property>
   </widget>
   <widget class="QWidget" name="layoutWidget">
    <property name="geometry">
     <rect>
      <x>30</x>
      <y>320</y>
      <width>166</width>
      <height>31</height>
     </rect>
    </property>
    <layout class="QHBoxLayout" name="horizontalLayout">
     <item>
      <widget class="QPushButton" name="play_tab">
       <property name="text">
        <string>Play</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QPushButton" name="pushButton_cancel_sort">
       <property name="text">
        <string>Cancel</string>
       </property>
      </widget>
     </item>
    </layout>
   </widget>
   <widget class="QWidget" name="layoutWidget">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>220</y>
      <width>184</width>
      <height>43</height>
     </rect>
    </property>
    <layout class="QHBoxLayout" name="horizontalLayout_2">
     <item>
      <layout class="QVBoxLayout" name="verticalLayout">
       <item>
        <widget class="QLabel" name="label_no_compare">
         <property name="styleSheet">
          <string notr="true">font: 63 9pt &quot;Yu Gothic UI Semibold&quot;;
color:white;</string>
         </property>
         <property name="text">
          <string>Number of Comparisions</string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QLCDNumber" name="lcdNumber_comaprision">
         <property name="styleSheet">
          <string notr="true">color: white;

</string>
         </property>
        </widget>
       </item>
      </layout>
     </item>
     <item>
      <layout class="QVBoxLayout" name="verticalLayout_2">
       <item>
        <widget class="QLabel" name="label_speed">
         <property name="styleSheet">
          <string notr="true">font: 63 9pt &quot;Yu Gothic UI Semibold&quot;;
color:white;</string>
         </property>
         <property name="text">
          <string>Animation Speed</string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QComboBox" name="speed_comboBox">
         <property name="styleSheet">
          <string notr="true">background-color: rgb(243, 243, 243);
selection-background-color: rgb(83, 83, 83);</string>
         </property>
        </widget>
       </item>
      </layout>
     </item>
    </layout>
   </widget>
   <widget class="QPushButton" name="enter_cip">
    <property name="geometry">
     <rect>
      <x>130</x>
      <y>140</y>
      <width>41</width>
      <height>20</height>
     </rect>
    </property>
    <property name="text">
     <string>Enter</string>
    </property>
   </widget>
   <widget class="QPushButton" name="enter_seql">
    <property name="geometry">
     <rect>
      <x>130</x>
      <y>190</y>
      <width>41</width>
      <height>20</height>
     </rect>
    </property>
    <property name="text">
     <string>Enter</string>
    </property>
   </widget>
  </widget>
  <widget class="QGraphicsView" name="graphicsView">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>10</y>
     <width>341</width>
     <height>281</height>
    </rect>
   </property>
  </widget>
  <widget class="QPlainTextEdit" name="info_text">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>300</y>
     <width>341</width>
     <height>111</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <family>Arial Rounded MT Bold</family>
     <pointsize>20</pointsize>
    </font>
   </property>
   <property name="styleSheet">
    <string notr="true">background-color: rgb(188, 188, 188);</string>
   </property>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
